### ISSUE TITLE
[feature request] use revision number(s) for all static assets?

### ISSUE THREAD

**Issue by turtleDev** [ravisuhag/jolly#12]
Only CSS files get revision in the current snapshot. However, other assets (especially JS) could also make use of it.

The reason I ask is because browsers can then cache your assets (by setting expiry headers), thereby reducing load times _considerably_. 

We could probably also extend this to images too.

**Comment by ravisuhag** [comment]
Yeah. That sounds great. We can surely go ahead with JS. Images might be an overkill.

**Comment by shruthi93** [comment]
Not able to save the data in mongodb.. What could be the possible reason for this issue. The output is showing the data is saved successfully but if I check in mongodb no data is present.

**Comment by ravisuhag** [comment]
@shruthi93 Would need more context to answer it. Also this question does not relate to this issue, so please create a separate issue for the same.

**Comment by OmerTurann** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.
edo1z, @br0ken1, @greyblake, @qiuzhuangshandian, @Sunt-ing, @Mgerxiu, @jjmatcha, @ssamkkim, @hyunchel, @SirDre, @PinYu1618, @aapsi, @massun-onibakuchi, @codingluan, @mailaneel, @BrigidBritney9, @liqunn, @lihuanghai, @backendcloud, @bransorem
